lib/import-article-from-medium.js

```diff
@@ -398,7 +398,7 @@ module.exports.getMarkdownFromLocalPost = async (
       localDom.querySelector('.p-name') || { textContent: '' }
     ).textContent.trim() || `Untitled Draft ${++untitledCounter}`
 
-  const slug = title ? slugify(title).toLowerCase() : redirect || ''
+  const slug = title ? slugify(title, {remove: /[\/:?*<">|.]/g, lower: true}): redirect || ''
```

Comment:
Can you explain this? maybe add a comment for the regex?

Reply:
**remove** option for removing special character from a title which may not allow for making directory. 
in my case
windows not allowing [  \ / : ? * < " > | ] these character set
and .(dot) for avoiding dot ended files/folder which[ make delete headche](https://ourcodeworld.com/articles/read/465/how-to-remove-folders-or-files-whose-name-ends-with-dots-in-windows) 
and lower option for lowercase instead of toLowerCase()

Reply:
so the default of the remove option is `/[^\w\s$*_+~.()'"!\-:@]/g`

I think we should keep that and add what's missing from your set, eg. ` \ /`, `?`, `<`, `>`, `|`.

Reply:
with `^`  return parsing error

working with `\w\s` without `^`  /[\w\s$*_+~.()'"!\-:@\/?]/g
![image](https://user-images.githubusercontent.com/18244052/62659788-f98df900-b989-11e9-8e8c-5511cf1b1b1a.png)
 
my working final is 
```javascript
// remove special character for avoid mkdir issue
const directoryRegex =  /[$*_+~.()'"!\-:@\/?]/g
```
...
```javascript
 const slug = title ? slugify(title, {remove: directoryRegex, lower: true}): redirect || ''
```

Reply:
I just tried `var r = /[^\w\s$*_+~.()'"!\-:@\/?]/g` in the node repl and it works fine. Please keep it as it's probably covering some edge cases we aren't thinking about

Reply:
the default `/[^\w\s$*_+~.()'"!\-:@]/g` regex has allowed character set 
so we don't want to add. we just removed which not allowed from my set
eg `*`, and `:`

so the regex should be `/[^\w\s$_+~.()'!\-@]/g`

Reply:
AH right, I missed the `^` at the beginning of the set :+1: